bugsnag-plugin-android-anr/src/main/jni/anr_handler.c

```diff
@@ -236,10 +224,7 @@ static void notify_anr_detected() {
     check_and_clear_exc(env);
   }
 
-  if (should_detach) {
-    (*bsg_jvm)->DetachCurrentThread(
-        bsg_jvm); // detach to restore initial condition
-  }
+  (*bsg_jvm)->DetachCurrentThread(bsg_jvm);
```

Comment:
Could we not run into issues where current thread was already attached, and our `AttachCurrentThread` was a no-op?